CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+2.1.1
```

Comment:
have unreleased there instead of the version

Reply:
Changed.